.circleci/config.yml

```diff
@@ -212,3 +271,13 @@ workflows:
           requires:
             - hold-native-tests
 
+      # Build Android release only on tags
+      - release-android:
+          requires:
+            - test-android
```

Comment:
since the `release-android` task rebuilds the bundle from scratch, I would run it just after `test-js-node-8`, only on `master`, what do you think?